adapters/nextmillennium/nextmillennium.go

```diff
@@ -0,0 +1,162 @@
+package nextmillennium
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type NextMillenniumBidRequest struct {
+	ID   string `json:"id"`
+	Test uint8  `json:"test,omitempty"`
+	Ext  struct {
+		Prebid struct {
+			StoredRequest struct {
+				ID string `json:"id"`
+			} `json:"storedrequest"`
+		} `json:"prebid"`
+	} `json:"ext"`
+}
+
+//MakeRequests prepares request information for prebid-server core
+func (adapter *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	resImps, err := getImpressionsInfo(request.Imp)
```

Comment:
As this function only returns the imp ext for all imps in the request, that's what you're passing into the `buildAdapterRequest` function which means you're pretty much throwing away all the imp info except for the imp ext. I am not sure how can your bidder service process the request just with the `placement_id` and not the rest of the imp info. Perhaps this wasn't intentional?

Reply:
At the moment we have all the information about who will use this placement_id. Thus, for now we leave everything as it is